.github/workflows/site.yaml

```diff
@@ -48,27 +51,33 @@ jobs:
         shell: Rscript {0}
 
       - name: Upload built site artifact
-        uses: actions/upload-artifact@v1
+        uses: actions/upload-pages-artifact@v3
         with:
-          name: site
           path: _book
 
       - name: Test website
         uses: docker://klakegg/html-proofer:3.18.8
```

Comment:
without knowing anything about this specifically, I wonder whether this is still fine or could be updated easily?